src/core/function_elt.tf

```diff
@@ -74,13 +74,31 @@ module "function_elt" {
     TARGETKAFKA_idempotent          = "true"
     TARGETKAFKA_transactionalId     = "IO_ELT"
     TARGETKAFKA_topic               = "io-cosmosdb-services"
-    ERROR_STORAGE_ACCOUNT           = module.storage_account_elt.name
-    ERROR_STORAGE_KEY               = module.storage_account_elt.primary_access_key
-    ERROR_STORAGE_TABLE             = azurerm_storage_table.fnelterrors.name
-    COMMAND_STORAGE                 = module.storage_account_elt.primary_connection_string
-    COMMAND_STORAGE_TABLE           = azurerm_storage_table.fneltcommands.name
-    IMPORT_TOPIC_NAME               = "import-command"
-    IMPORT_TOPIC_CONNECTION_STRING  = module.event_hub.keys["import-command.io-fn-elt"].primary_connection_string
+
+    MESSAGES_TOPIC_NAME              = "pdnd-io-cosmosdb-messages"
+    MESSAGES_TOPIC_CONNECTION_STRING = module.event_hub.keys["io-cosmosdb-services.io-fn-elt"].primary_connection_string
+    MESSAGES_LEASES_PREFIX           = "messages-001"
+
+    MESSAGE_STATUS_TOPIC_NAME              = "pdnd-io-cosmosdb-message-status"
+    MESSAGE_STATUS_TOPIC_CONNECTION_STRING = module.event_hub.keys["io-cosmosdb-services.io-fn-elt"].primary_connection_string
+    MESSAGE_STATUS_LEASES_PREFIX           = "message-status-001"
+
+    NOTIFICATION_STATUS_TOPIC_NAME              = "pdnd-io-cosmosdb-message-status"
+    NOTIFICATION_STATUS_TOPIC_CONNECTION_STRING = module.event_hub.keys["io-cosmosdb-services.io-fn-elt"].primary_connection_string
```

Comment:
ditto

Reply:
Fixed in [d665f2f](https://github.com/pagopa/io-infra/pull/136/commits/d665f2f4fc9d9359a337a2fcea58e95b7675340d)